src/coreclr/gc/gcpriv.h

```diff
@@ -266,6 +266,7 @@ void GCLog (const char *fmt, ... );
 // to do so.
 //#define dprintf(l, x)
 #define dprintf(l,x) STRESS_LOG_VA(l,x);
+//#define dprintf(l,x) {if ((l <= 2) || (l == REGIONS_LOG)) {STRESS_LOG_VA(l,x);}}
```

Comment:
>//#define dprintf(l,x) [](http://example.com/codeflow?start=0&length=22)

get rid of this before you checkin? I also don't mind if you change the existing dprintf to this

Reply:
I got rid of the line.